### PR TITLE
prevent error on catalog missing from config

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
## What does this PR change?
* Fix issue with Athena configuration where new catalog parameter was required when unmarshalling json of config. This bug causes existing configs to fail to load

## Does this PR relate to any other PRs?
* https://github.com/opencost/opencost/pull/2141

## How will this PR impact users?
no impact to user

## Does this PR address any GitHub or Zendesk issues?
* Closes ...

## How was this PR tested?
* Tested by loading configs missing this value

## Does this PR require changes to documentation?
* 

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
*

### CODE REVIEW

File: pkg/cloud/aws/athenaconfiguration.go [hunk 1]
```diff
@@ -149,11 +149,13 @@ func (ac *AthenaConfiguration) UnmarshalJSON(b []byte) error {
 	}
 	ac.Database = database
 
-	catalog, err := config.GetInterfaceValue[string](fmap, "catalog")
-	if err != nil {
-		return fmt.Errorf("AthenaConfiguration: UnmarshalJSON: %s", err.Error())
+	if _, ok := fmap["catalog"]; ok {
+		catalog, err := config.GetInterfaceValue[string](fmap, "catalog")
+		if err == nil {
```
Comment (michaelmdresser) [inline pkg/cloud/aws/athenaconfiguration.go]: This should be `!=`?

Comment (nikovacevic) [reply pkg/cloud/aws/athenaconfiguration.go]: ^^

File: pkg/cloud/aws/athenaconfiguration.go [hunk 2]
```diff
@@ -133,43 +133,45 @@ func (ac *AthenaConfiguration) UnmarshalJSON(b []byte) error {
...
 	}
 	ac.Database = database
 
-	catalog, err := config.GetInterfaceValue[string](fmap, "catalog")
-	if err != nil {
-		return fmt.Errorf("AthenaConfiguration: UnmarshalJSON: %s", err.Error())
+	if _, ok := fmap["catalog"]; ok {
+		catalog, err := config.GetInterfaceValue[string](fmap, "catalog")
+		if err != nil {
+			return fmt.Errorf("AthenaConfiguration: UnmarshalJSON: %w", err)
```
Comment (Sean-Holcomb) [inline pkg/cloud/aws/athenaconfiguration.go:155]: @michaelmdresser you are slipping man, you didn't even comment on the unwrapped errors

Comment (michaelmdresser) [reply pkg/cloud/aws/athenaconfiguration.go:155]: 😆 ahh its a dark day isn't it

**Review by michaelmdresser — APPROVED** [review]

**Comment by a7i** [comment]
thank you for the fix @michaelmdresser 🏆

**Pull request merged**